### ISSUE TITLE
bug: can not forceThalesIntoRemoteScript

### ISSUE THREAD

**Issue by ilka-schulz** [Zahner-elektrik/Thales-Remote-Python#11]
I can not force my thales device into remote script mode:

```python
connection = ThalesRemoteConnection()
if not connection.connectToTerm("130.75.205.165"):
    raise Exception("could not connect to Thales Remote")
device = ThalesRemoteScriptWrapper(connection)
device.forceThalesIntoRemoteScript() # hangs forever
# further method calls are not executed
```

I tried this with the Thales XT 5.8.2 being in two different states, neither of it worked:
- My Zennium device is doing nothing. It is in `TCP/IP VI remote control` mode saying `running`.
- My Zennium device is doing nothing. It shows the "classic mode" start screen from where to choose different measurement techniques.

**Comment by maxkrapp1** [comment]
Unfortunately, I cannot reproduce the problem. I can start the remote with the following example.
https://github.com/Zahner-elektrik/Thales-Remote-Python/blob/main/Examples/BasicIntroduction/BasicIntroduction.ipynb
Please compare your code with the example.

You can also test to omit zahnerZennium.forceThalesIntoRemoteScript() and instead click the Zahner logo in the Thales window then select "Optional Methods" and "Remote Control". Then you can test if the control works in principle and only the remote start does not work.

**Comment by ilka-schulz** [comment]
It appears to be a duplicate of #13. The `Remote Control` GUI says `running` as soon as a client is connected. I the client does not connect cleanly, it just stays in that mode and rejects further attempts to call remote-control the device.

**Comment by maxkrapp1** [comment]
Yes this is unfortunately a known problem that it can happen that the Term comes into an unrecoverable state if you do not cleanly disconnect as mentioned in #13.

**Comment by maxkrapp1** [comment]
This was fixed with term version 5.8.8.
